hera_pspec/pspecdata.py

```diff
@@ -3244,7 +3248,7 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
 
                     # Calculate unnormalized bandpowers
                     if verbose: print("  Building q_hat...")
-                    qv = self.q_hat(key1, key2, exact_norm=exact_norm, pol=pol)
+                    qv = self.q_hat(key1, key2, exact_norm=exact_norm, pol=pol, allow_fft=allow_fft)
```

Comment:
`get_Q_alt` returns the same matrix regardless of whether `allow_fft` is set though I agree we should propagate it there as well to avoid confusion.

Reply:
Looking some more `get_Q_alt` does seem to have internal checks that avoid ever using the fft unless `spw_Ndlys == nfreq`. I think this should be fine even if we have `sampling=False` since `Q_alt` is defined to be with sampling (infinitesimal fourier bins).